rdkit/Dbase/DbModule.py

```diff
@@ -11,17 +11,17 @@
 from rdkit import six
 from rdkit import RDConfig
 
```

Comment:
There's enough cruft from old versions of the RDKit and no-longer-existent databases/database wrappers in this code that I think it's worth considering getting rid of most of it. I will take a look at feasibility of that.